pkgs/top-level/all-packages.nix

```diff
@@ -18201,7 +18201,9 @@ with pkgs;
 
   qbittorrent = libsForQt5.callPackage ../applications/networking/p2p/qbittorrent { };
 
-  qcomicbook = libsForQt5.callPackage ../applications/graphics/qcomicbook { };
+  qcomicbook = libsForQt5.callPackage ../applications/graphics/qcomicbook {
+    poppler = libsForQt5.poppler;
```

Comment:
This does not have any effect and can be removed. Because of  `qcomicbook = libsForQt5.callPackage ...`, `poppler` is already the qt5 variant.